mlflow/pipelines/regression/v1/pipeline.py

```diff
@@ -185,6 +185,16 @@ class RegressionPipeline(_BasePipeline):
         RegisterStep,
     )
 
+    def _get_default_step_class(self) -> Type[BaseStep]:
+        return RegisterStep
+
+    def _get_connected_dag(self, target_step) -> List[BaseStep]:
+        index = self._steps.index(target_step)
+        delimiter_index = self._PIPELINE_STEPS.index(IngestStep)
+        if index < delimiter_index:
+            return self._steps[:delimiter_index]
+        return self._steps[delimiter_index:]
```

Comment:
Can we be a bit more explicit here by separating `_PIPELINE_STEPS` into `_TRAIN_DAG_STEPS`, `_SCORE_DAG_STEPS` and then checking whether the `target_step` is in one of these lists or the other?